### DISCUSSION TITLE
multiple bucket policies are not allowed

### DISCUSSION THREAD

**Discussion by barabo** [terraform-linters/tflint-ruleset-aws#492] [Ideas]
If multiple bucket policies are defined for a single bucket, they both cannot be applied.  One recommended approach is to create a policy with multiple statements and use that as the bucket policy.

Would it be hard to enable an error when a bucket has multiple bucket policies defined?

**Comment by wata727** [comment]
Looks good. Such a rule would be useful unless you have a good reason to tie multiple bucket policies together.
